plugins/services/src/js/reducers/serviceForm/JSONReducers/Containers.js

```diff
@@ -287,6 +295,17 @@ function containersParser(state) {
   }, []);
 }
 
+function deleteContainerImageCheck(image) {
```

Comment:
Also, this does behave different than the forcePull for single container services. Using just `if (!image.id) return true` would make it work that same!